src/archetypes/referencebrowserwidget/skins/referencebrowser/referencebrowser.js

```diff
@@ -447,6 +447,17 @@ jQuery(function ($) {
         });
     }
 
+    $(document).on('change', '#indexSelector', function (event) {
+        if ($.inArray(this.value, eval($('#searchWildcardHelp').data()['wildcardableindexes'])) === -1) {
```

Comment:
I'm not a fan of eval for security reasons. jQuery.parseJSON may be better here.
But is it really necessary?
From https://api.jquery.com/data/
"When the data attribute is an object (starts with '{') or array (starts with '[') then jQuery.parseJSON is used to parse the string; it must follow valid JSON syntax including quoted property names. If the value isn't parseable as a JavaScript value, it is left as a string."